Add DevExtreme Versions Topic (#8470)
DevExpress/devextreme-documentation: concepts/Common/DevExtreme Versions/00 DevExtreme Versions.md

> DevExtreme does not use semantic versioning and follows the DevExpress [product delivery model](https://www.devexpress.com/support/delivery-model.xml). New versions are released twice per year, in December and June. Minor updates are released approximetly once per month, or more frequently, if required.

Reviewer: It's better to avoid mentioning exact months or say at least approximately or smth like this

---
title: DevExtreme Versions 
---

> DevExtreme does not use semantic versioning and follows the DevExpress product delivery model. See the following topic for more information: [Product Delivery](https://www.devexpress.com/support/delivery-model.xml).

Reviewer 2: Ray always edits references that simply say "topic". You can either expand it to "help topic" or say "article". Since it's a page on our website (not part of documentation), I'd rather go with "article". 

```suggestion
DevExtreme does not use semantic versioning and follows the DevExpress product delivery model. Review the following article for more information: [Product Delivery](https://www.devexpress.com/support/delivery-model.xml).
```

> To minimize security-related risks, you should always install and use the most recent version of DevExtreme available.

Reviewer 2: ```suggestion
To minimize security-related risks, you should always install and use the latest available DevExtreme version.
```
